### PR TITLE
Updated  bearssl to its latest version.

### PR DESCRIPTION
**Pull request by slaff**

### CODE REVIEW

File: Sming/Components/ssl/component.mk [hunk 1]
```diff
@@ -52,7 +52,7 @@ include/ssl/private_key.h:
 	$(info Generating unique certificate and key. This may take some time...)
 	$(Q) mkdir -p $(SSL_INCLUDE_DIR) $(SSL_CERT_DIR)
 	$(Q) chmod a+x $(SSL_TOOLS_PATH)/make_certs.sh
-	cd $(SSL_CERT_DIR); SSL_INCLUDE_DIR=$(SSL_INCLUDE_DIR) $(SSL_TOOLS_PATH)/make_certs.sh
+	$(Q) cd $(SSL_CERT_DIR); SSL_INCLUDE_DIR=$(SSL_INCLUDE_DIR) $(SSL_TOOLS_PATH)/make_certs.sh; rm -rf $(SSL_CERT_DIR)
```
Comment (mikee47) [inline Sming/Components/ssl/component.mk:55]: Some of those files may need to be kept; How about using `out/ssl/cert` instead?

**Review by mikee47 — APPROVED** [review]
Looks good. I've moved the openssl fix from #2224 as it occurs running openssl in all architectures.

**Comment by mikee47** [comment]
Just one other thing. I get `rm: cannot remove 'srl': No such file or directory` when certs are generated. In https://github.com/SmingHub/Sming/blob/develop/Sming/Components/ssl/Tools/make_certs.sh#L151 is this cleanup actually necessary?

**Comment by slaff** [comment]
> Just one other thing. I get rm: cannot remove 'srl': No such file or directory when certs are generated.

I will change this to `rm *.srl`. Openssl keeps the serial number per certificate in x.srl. 

> is this cleanup actually necessary?

Well... these files are intermediate so they can be removed.

**Pull request merged**